src/locales/en-US/commands.json

```diff
@@ -1064,42 +1066,42 @@
     }
   },
   "anal": {
-    "commandDescription": "Fetches a random image from r/anal and displays it."
+    "commandDescription": "Shows a random image from r/anal."
   },
   "ass": {
-    "commandDescription": "Fetches a random image from r/ass and displays it."
+    "commandDescription": "Shows a random image from r/ass."
   },
   "boobs": {
-    "commandDescription": "Fetches a random image from r/boobs and displays it."
+    "commandDescription": "Shows a random image from r/boobs."
   },
   "porn": {
-    "commandDescription": "Fetches a random image from r/porn and displays it."
+    "commandDescription": "Shows a random image from r/porn."
   },
   "porngifs": {
-    "commandDescription": "Fetches a random image from r/porngifs and displays it."
+    "commandDescription": "Shows a random image from r/porngifs."
   },
   "pussy": {
-    "commandDescription": "Fetches a random image from r/pussy and displays it."
+    "commandDescription": "Shows a random image from r/pussy."
   },
   "yaoi": {
-    "commandDescription": "Fetches a random image from r/yaoi and displays it."
+    "commandDescription": "Shows a random image from r/yaoi."
   },
   "yiff": {
-    "commandDescription": "Fetches a random image from r/yiff and displays it. (A.K.A Furry Porn)"
+    "commandDescription": "Shows a random image from r/yiff. (A.K.A Furry Porn)"
   },
   "yuri": {
-    "commandDescription": "Fetches a random image from r/yuri and displays it."
+    "commandDescription": "Shows a random image from r/yuri."
   },
   "welcometranslator": {
     "commandDescription": "Welcomes a new translator.",
     "commandUsage": "<user>",
-    "noMember": "You need to mention someone!",
+    "noMember": "You have to mention someone!",
     "cantWelcomeYourself": "You can't welcome yourself!",
-    "welcomeMessage": "Welcome to the team {{member}}!\nIn <#447005926981894144> you can talk about translation related stuff.\nYou will see announcements in <#476110480377905163>.\nYou will get pinged in <#450050899197296651> when there are new strings.\nLast but not least, you can see some useful info in <#447962790918094878>.\n\nIf you have any questions, feel free to ask!"
+    "welcomeMessage": "Welcome to the team {{member}}!\nIn <#447005926981894144> you can talk about translation related stuff.\nYou will see announcements in <#476110480377905163>.\nYou will get pinged in <#450050899197296651> when there are new strings.\nLast but not least, you can see some useful info in <#447962790918094878>.\n\nIf you have any question, feel free to ask!"
```

Comment:
I think this should stay as „questions”, for the reason that a user could have multiple questions.

Reply:
Yep, I read up a little bit on that grammar. After `any`, plural forms should be used in most cases. Fix coming in next commit.